.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
```

Comment:
For linter error: Could it be because you removed this 👇 
```
  "rules": {
    "no-shadow": "off",
    "no-param-reassign": "off",
    "eol-last": "off",
    "arrow-parens": "off",
  }
```

We shouldn't edit the `eslintrc.json` file from Microverse.

Reply:
> Yes @bellom thats the one we used in our project and we didn't change it.

Also, check this out for the linter error; 
https://eslint.org/docs/2.0.0/rules/no-unreachable

Reply:
Thanks for the comment, but like I said we didn't remove any code. It seems that we had to run npm in the project before running npx eslint - - init and the liners were not properly installed. We will review the process and check the errors

Reply:
Oh, I wouldn't know right;
As you can see https://user-images.githubusercontent.com/31897434/72831785-d6c77680-3c83-11ea-9012-d256949782fd.png